infrastructure/repository/labels-workflow.tf

```diff
@@ -117,9 +125,9 @@ variable "workflow_labels" {
       color       = "828a90", # color:stale grey
       description = "Repository modifications; GitHub Actions, developer docs, issue templates, codeowners, changelog."
     },
-    "roadmap" = {
+    "prioritized" = {
```

Comment:
My only concern with removing `roadmap` before we get `prioritized` in place is that it'll remove that label from everything that currently has it. I almost wonder if we should do it in two passes so that we can make sure everything that has `roadmap` currently gets the `prioritized` label before we remove it. Other than that, this looks good to me

Reply:
Theres only 2 open ones and I've made a note of them. That said, it doesn't make sense to have `prioritized` on closed issues so we should add removing that label on merge/release to the automation list